.circleci/config.yml

```diff
@@ -230,7 +230,7 @@ jobs:
           command: ./gradlew assembleSelfSignedRelease
 
       - run:
-          name: Check APK size isn't larger than 11.5MB
+          name: Check APK size isn't larger than 11.7MB
```

Comment:
Yes, I'm not sure if it's only because of Mapbox. To make sure I would need to check the size after updating every single dependency however the difference (0.2MB) wasn't big enough to make me do that.